Nimb3s.Automaton.DataAccess/JobData.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Nimb3s.Automaton.Pocos;
+using Nimb3s.Automaton.Pocos.Models;
+using System;
```

Comment:
your db call will be something like this

```
            return await connection
                .QuerySingleAsync<JobStatusDetailsEntity >(sql: $"{Schema}.p_Get{entityName}DetailsBy{nameof(jobId)}", param: dp, commandType: CommandType.StoredProcedure, transaction: transaction)
                .ConfigureAwait(false);
```

you'll want to place your new stored procedure in the [Nimb3s.Automaton.Database project](https://github.com/nimb3s/automaton/tree/main/src/Nimb3s.Automaton.Database/Job/Stored%20Procedures). you can use the [Job.p_GetAllWorkItemsByJobId](https://github.com/nimb3s/automaton/blob/main/src/Nimb3s.Automaton.Database/Job/Stored%20Procedures/p_GetAllWorkItemsByJobId.sql) procedure as an example. just make sure you add the sproc to the project by right clickign the project and adding a new stored procedure